src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx

```diff
@@ -33,7 +33,7 @@ const PlanDefinitionList = (props: PlanListProps) => {
   const { fetchPlans, plans, service } = props;
   const [loading, setLoading] = useState<boolean>(true);
 
-  const apiService = new service('/plans');
+  const apiService = new service('plans');
```

Comment:
`export const OPENSRP_PLANS = 'plans';` This is trivial but maybe we should also use constants for the api endpoints.

Reply:
Agreed, let's move endpoint strings like this into constants :)